code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -177,7 +177,14 @@
 						makeNewConstruct(/mob/living/simple_animal/hostile/construct/wraith, A, user, 0, T.loc)
 
 					if("Artificer")
-						makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
+						if(iscultist(user))
+							makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
+
+						if(iswizard(user))
+							makeNewConstruct(/mob/living/simple_animal/hostile/construct/builder, A, user, 0, T.loc)
```

Comment:
Don't copy-paste. Use `if(iscultist(user) || iswizard(user))` (logical OR) instead.
Also, this will probably cause conflicts for datum antags, I think? I'll let @Incoming5643 chime in here.